orga/translations/en.json

```diff
@@ -1,5 +1,16 @@
 {
   "current-lang": "en",
+  "common": {
+    "pagination": {
+      "result-by-page": "See",
+      "page-number": "Page {current} / {total}",
+      "action": {
+        "select-page-size": "Select the number of items by page",
+        "previous": "Go to previous page",
+        "next": "Go to next page"
+      }
+    }
+  },
```

Comment:
J'ai inventé ces traductions car elles ne sont pas dans le fichier.

Reply:
J'ai un doute sur "Select the number of items by page" mais sinon le reste 👍 (le français est pas top d'ailleurs ^^)